packages/blade/src/components/Amount/amountTokens.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
why?

Reply:
We have not fixed Amount component. We will be picking up during individual component migration since its a bit more complex and needs other rework as well.

Making it ts-nocheck because otherwise the typescript fails and we don't get chromatic published to test out changes.